src/components/MarkdownProvider/components/CodeExample/utils/retrieveCodesandboxParameters.ts

```diff
@@ -64,6 +64,7 @@ const packageJson = `
     "@zendeskgarden/react-breadcrumbs": "^8.x",
     "@zendeskgarden/react-buttons": "^8.x",
     "@zendeskgarden/react-chrome": "^8.x",
+    "@zendeskgarden/react-datepickers": "^8.x",
```

Comment:
Please also add `date-fns` in here as now it is a direct dependency within some of the examples.

Reply:
Should I add it to the package.json as well?

Reply:
Nope.

Reply:
Yes, probably should.